overlore/jsonrpc/methods/spawn_npc/constants.py

```diff
@@ -1,3 +1,14 @@
-NPC_PROFILE_SYSTEM_STRING = """Imagine you're the game master for a fantasy strategy game, you are tasked with creating a Non-Playable Character."""
+NPC_PROFILE_SYSTEM_STRING = """Imagine you're the game master for a fantasy strategy game, you are tasked with creating the full name, backstory and a character trait for a Non-Playable Character.
+Output format example:
+{
+    full_name: "value",
+    character_trait: "value",
+    background: "value"
+}
+"""
 
-NPC_PROFILE_USER_STRING = """Generate an villager with a {trait_type} trait"""
+NPC_PROFILE_USER_STRING = """Generate for a villager a {trait_type} trait, a backstory and a full name. These are the characteristics of this villager:
```

Comment:
Are the results better/worse/the same as before with this change?

Reply:
Pretty much the same, difference being that we don't generate the characteristics within the GPT prompt anymore.